rocketpy/utilities.py

```diff
@@ -229,52 +229,51 @@ def fin_flutter_analysis(
     ------
     None
     """
+    found_fin = False
 
-    # First, we need identify if there is at least a fin set in the rocket
-    for aero_surface in flight.rocket.aerodynamic_surfaces:
+    # First, we need identify if there is at least one fin set in the rocket
+    for aero_surface in flight.rocket.fins:
         if isinstance(aero_surface, TrapezoidalFins):
             # s: surface area; ar: aspect ratio; la: lambda
             root_chord = aero_surface.root_chord
             s = (aero_surface.tip_chord + root_chord) * aero_surface.span / 2
             ar = aero_surface.span * aero_surface.span / s
             la = aero_surface.tip_chord / root_chord
-
-    # This ensures that a fin set was found in the rocket, if not, break
-    try:
-        s = s
-    except NameError:
-        print("There is no fin set in the rocket, can't run a Flutter Analysis.")
+            if not found_fin:
+                found_fin = True
+            else:
+                warnings.warn("More than one fin set found. The last one will be used.")
+    if not found_fin:
+        print("There is no TrapezoidalFins in the rocket, can't run Flutter Analysis.")
```

Comment:
Should't we raise an `AttributeError` here?

Reply:
That's a good suggestion, way better than a print statement.
But the warning above must be kept, bc the code could still work.

Reply:
Solved here: [6f4c3c3](https://github.com/RocketPy-Team/RocketPy/pull/510/commits/6f4c3c306f1744821f640b3a9b07e4584d946a75)